src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from 'react';
+import { Calculator, exceedLimitOfDigitError } from './Model/Calculator';
+
+const BUTTONS = {
+  DIGIT: ['9', '8', '7', '6', '5', '4', '3', '2', '1', '0'],
+  ALL_CLEAR: 'AC',
+  OPERATOR: ['/', 'X', '-', '+', '='],
+};
+
+const App = () => {
+  const [calculator, setCalculator] = useState(Calculator());
```

Comment:
```suggestion
  const [calculator, setCalculator] = useState(() => Calculator());
```

useState의 initialState가 식의 결과라면 위와 같이 함수로 전달하는 것이 권장됩니다~
https://reactjs.org/docs/hooks-reference.html#lazy-initial-state